hypha/apply/projects/permissions.py

```diff
@@ -34,7 +34,7 @@ def can_approve_contract(user, project, **kwargs):
     if not user.is_authenticated:
         return False, "Login Required"
 
-    if user.is_apply_staff and not user.is_contracting and not user.is_applicant:
+    if user.is_apply_staff:
         return True, "Only Staff can approve the contract"
```

Comment:
If removing `is_applicant` then maybe ensure that the user approving is not applicant or partner on the project? I am assuming it's going be rare but just thinking around _least privilege_ point of view.